maths/area.py

```diff
@@ -458,6 +458,52 @@ def area_rhombus(diagonal_1: float, diagonal_2: float) -> float:
     return 1 / 2 * diagonal_1 * diagonal_2
 
 
+def area_kite(diagonal_1: float, diagonal_2: float) -> float:

```

Comment:
@cclauss  need you opinion on something this function is quite similar to the one above like exactly the same just with different tests and a different name are similar functions allowed?

Reply:
I think that if the function above will also work for kites then we should add a comment to that function and avoid a duplicate function.

Reply:
I completely agree.